File: backend/Nexos.Domain/Entities/Paciente.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Nexos.Domain.Entities
{
    public class Paciente : Entity
    {
        [Required]
        public string Nombre { get; set; }

        [Required]
        public string Apellido { get; set; }
        
        [Required]
        public string SeguroSocial { get; set; }
        
        public string CodigoPostal { get; set; }
        
        public string Telefono { get; set; }
        
        public virtual ICollection<Doctor> Doctores { get; set; }
    }
}

File: frontend/src/app/components/pacientes/paciente.ts
export class Paciente{
    constructor(
        public id:string,
        public nombre:string,
        public apellido:string,
        public seguroSocial:string,
        public codigoPostal:string,
        public telefono:string
    ){}
}

export class Doctor{
    constructor(
        public id:string,
        public nombre:string,
        public apellido:string,
        public especialidad:string,
        public hospital:string
    ){}
}